src/test/java/io/openliberty/tools/intellij/it/GradleSingleModMPProjectTest.java

```diff
@@ -88,6 +96,22 @@ public String getWLPInstallPath() {
         return WLP_INSTALL_PATH;
     }
 
+    /**
+     * This method expands specific nodes within a project tree represented by a JTreeFixture.
+     * It locates the ProjectFrameFixture using a remote robot and retrieves a JTreeFixture for the project view.
+     * Then, it expands the nodes corresponding to the given path within the project tree.
+     *
+     * @return JTreeFixture representing the expanded project tree nodes.
+     */
+    @Override
+    public JTreeFixture expandProjectTreeNodes() {
```

Comment:
We are not using the returned value, so we can make the return type to void.

Reply:
Removed this method to implement it in another way